### DISCUSSION TITLE
keepassXC errors

### DISCUSSION THREAD

**Discussion by welshkyle123** [runelite/runelite#16615] [Q&A]
"abex.os.keepassxc.proto.KeePassEx..." <-- this is the error runelite gives me

hi guys, sorry if this isn't the right place but i'm not sure where to get help about this. keepassxc is  supposed to pull my passwords from my password file on my desktop but runelite keeps giving me this error. it works fine on my desktop but not on my laptop

any suggestions are appreciated

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
